modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/type/EmailNotificationType.java

```diff
@@ -343,6 +344,12 @@ public void validateNotificationTemplate(
 		if (Validator.isNull(notificationRecipientSettingsMap.get("from"))) {
 			throw new NotificationTemplateFromException("From is null");
```

Comment:
@murilostodolni replace here with the use of NotificationRecipientSettingException please. Thx.